acoustic_msgs/msg/SonarRanges.msg

```diff
@@ -15,31 +15,7 @@ std_msgs/Header header
 # than one thing to flag beams as bad. To check if a beam is good, simply
```

Comment:
(comment for line 5; it wouldn't let me select it ...)

Do you want to add attributions to the other institutions who have worked on this, or just delete this line?
(I'm ambivalent, but as it stands now, it's inaccurate)